core/controllers/admin.py

```diff
@@ -188,7 +188,7 @@ def get(self):
         feature_flag_dicts = feature_services.get_all_feature_flag_dicts()
 
         config_properties = config_domain.Registry.get_config_property_schemas()
-        # Removes promo-bar related configs as promo-bar is handlded by
+        # Removes promo-bar related configs as promo-bar is handled by
```

Comment:
I had a chat with other team members and we thought about whether we can have such PRs and we think that it's fine to have a separate PR to improve the codebase (it can be a minor spell change as well and yeah, thanks for catching this and creating a PR :))